packages/in17n/package.json

```
@@ -0,0 +1,72 @@
+{
+	"name": "@sapphire/plugin-in17n",
+	"version": "1.0.0",
+	"description": "> TODO: description",
+	"author": "@sapphire",
+	"maintainers": [
+		"QuantumlyTangled <yo@quantumlytangled.com>"
+	],
+	"contributors": [
+		"QuantumlyTangled <yo@quantumlytangled.com>"
+	],
+	"license": "BSD-3-Clause",
+	"main": "dist/index.js",
+	"homepage": "https://github.com/sapphire-project/plugins/tree/main/packages/in17n",
+	"scripts": {
+		"lint": "eslint src tests --ext ts --fix",
+		"build": "tsc -b src",
+		"postbuild": "run-p esm:**",
+		"esm:default": "gen-esm-wrapper dist/index.js dist/index.mjs"
+	},
+	"dependencies": {
+		"@sapphire/utilities": "^1.3.0",
+		"i18next": "^19.8.4",
+		"i18next-fs-backend": "^1.0.7"
+	},
+	"peerDependencies": {
+		"@sapphire/framework": "1.0.0-alpha.1",
+		"@sapphire/pieces": "1.0.1",
+		"@types/i18next-fs-backend": "^1.0.0",
+		"discord.js": "^12.5.1"
+	},
+	"peerDependenciesMeta": {
+		"@types/i18next-fs-backend": {
+			"optional": true
+		},
+		"discord.js": {
+			"optional": true
+		}
```

```suggestion
		}
```

`src/lib/extensions/In17nMessage.ts` imports DJS stuff (and it cannot be a type import due to `Structures`). If you don't have DJS installed then TS will complain about that import, given you don't provide `--skipLibCheck` (which one shouldn't anyway)